Makefile

```diff
@@ -188,6 +188,14 @@ test/addons/.buildstamp: config.gypi \
 # TODO(bnoordhuis) Force rebuild after gyp update.
 build-addons: $(NODE_EXE) test/addons/.buildstamp
 
+ifeq ($(OSTYPE),$(filter $(OSTYPE),darwin aix))
+  XARGS = xargs
+else
+  XARGS = xargs -r
+endif
+clear-stalled:
+	ps awwwx | grep Release/node | grep -v grep | awk '{print $$1}' | $(XARGS) kill
```

Comment:
I think you could just add the same line up to the awk above to list the processes that will be killed, i.e.:

```bash
ps awwwx | grep Release/node | grep -v grep
ps awwwx | grep Release/node | grep -v grep | awk '{print $$1}' | $(XARGS) kill
```